_layouts/service.html

```diff
@@ -3,13 +3,17 @@
 ---
 <div class="page-header">
   <h2 class="text-center">
+    {% if page.pictures.icon and page.pictures.icon != empty  %}
```

Comment:
I don't think you need the empty check here. That was in other if checks for empty arrays